.bazelci/presubmit.yml

```diff
@@ -27,15 +27,15 @@ tasks:
     platform: ubuntu2004
     shell_commands:
     # Install xmllint
-    - sudo apt update && sudo apt install -y libxml2-utils
+    - sudo apt update && sudo apt install --reinstall libxml2-utils -y
```

Comment:
Due to https://buildkite.com/bazel/rules-scala-scala/builds/4919#01915033-1744-4437-9781-756df196c887/122-496